haskell/test-common/Lorentz/Contracts/Test/Common.hs

```diff
@@ -62,6 +63,8 @@ commonOperator = genesisAddress6
 testOwner = genesisAddresses !! 7
 testPauser = genesisAddresses !! 8
 testMasterMinter = genesisAddresses !! 9
+testIssuer = testOwner
+testAdmin = testOwner
```

Comment:
Apparently `testIssuer` and `testAdmin` are used only in `Main`, so I wouldn't put them into `Common`.